query-node/schemas/contentNftEvents.graphql

```diff
@@ -399,6 +402,9 @@ type NftBoughtEvent implements Event @entity {
 
   "Member that bought the NFT."
   member: Membership!
+
+  "Price for which NFT was bought"
+  price: Int!
```

Comment:
I think we should add `price` property to `Auction` entity as well so we have information about the actual price NFT was sold in an auction. It should be simple change since you have this value (`boughtPrice`) in `finishAuction` function already.

Reply:
`Auction.price` may be a bit misleading. Do I understand correctly that this value would be populated only if the action has finished? If so, maybe some more descriptive name could be used, like `finalPrice` or something along the lines